content/rs/references/rlcheck.md

```diff
@@ -0,0 +1,29 @@
+---
+Title: Node Verification Utility (rlcheck)
+description:
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+rlcheck is a utility that runs various health checks on an Redis Enterprise Software (RS) node and alerts on any issues found.
+This utility can be useful to confirm a successful installation or to verify that the node is functioning properly.
+
+You can access rlcheck from the host command-line interface (CLI).
+The output of rlcheck shows information specific to the host that you run it on.
+
+To open the rladmin CLI:
+
+1. Login to the RS host with an account that is either:
+    - a member of the **redislabs** OS group
+    - change user to root with command: `sudo su --`
```

Comment:
@Paz-Yanover Remove this line?

Reply:
@bmansheim - can you please the history of this change? I recall we had some requests to had the sudo. If there is no relevant history (and explanations), let's remove it

Reply:
@Paz-Yanover I don't see any specific requests. I'll remove it.